CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+First of all, thank you for helping us, improve the Thanos!
+
+When contributing not obvious change to Thanos repository, please first
+discuss the change you wish to make via issue or slack, or any other
+method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```

Comment:
Please follow the (code of conduct)(link) in all your interactions with the project.